tests/lib/rules/space-unary-ops.js

```diff
@@ -123,6 +123,18 @@ ruleTester.run("space-unary-ops", rule, {
         {
             code: "++ foo",
             options: [{ "nonwords": true }]
+        },
+        {
+            code: "function *foo () { yield (0) }",
+            ecmaFeatures: { generators: true }
+        },
+        {
+            code: "function *foo() { yield +1 }",
+            ecmaFeatures: { generators: true }
+        },
+        {
+            code: "function *foo() { yield *0 }",
```

Comment:
Just to be sure, the following examples should be valid:

```js
function *foo() { yield* 0 }
function *foo() { yield * 0 }
```

and those invalid:

```js
function *foo() { yield*0 }
function *foo() { yield *0 }
```

:question: